CHANGELOG.md

```diff
@@ -0,0 +1,142 @@
+# Changelog
+
+
+## [1.0.5] - 2018-06-04
+### Fixed
+- Minor fixes
```

Comment:
Oh, come on :) Let's at least copy commit messages.

Reply:
for [1.0.5](https://github.com/DivanteLtd/vue-storefront/releases/tag/v1.0.5) and [1.0.0](https://github.com/DivanteLtd/vue-storefront/releases/tag/v1.0.0) the releases link showing same, I have added for `1.0.0`  but not for `1.0.5`